assets/scss/components/_header-bar.scss

```diff
@@ -339,6 +339,14 @@
         span {
           color: govuk-colour('black');
         }
+
```

Comment:
You don't need either of the `a` rules as we're changing it to the `span`, so the above rule will already style that correctly.

Reply:
@davidmiddletonmoj  Good spot - updated CSS and classes - ready for review.